### DISCUSSION TITLE
What are the exact validations that happens at "post-build validation" stage

### DISCUSSION THREAD

**Discussion by hamaney** [microsoft/vcpkg#22950] [Q&A] [answered]
I am interested to know what are the exact tests and validations performed by vcpkg to ensure that my port has built correctly?
`[cmake] -- Performing post-build validation`
any reference would be a plus! 

Thanks

**Comment by autoantwort** [comment, marked as answer]
https://github.com/microsoft/vcpkg-tool/blob/main/src/vcpkg/postbuildlint.cpp
